### DISCUSSION TITLE
Azure to GCP Storage transfer errors

### DISCUSSION THREAD

**Discussion by ajayra04** [skyplane-project/skyplane#624] [Q&A]
Hi Team,

We need to transfer the data from azure storage to GCP Buckets, we setup the tool skyplane and trying to use for this purpose.  

We are facing the challenges and getting below error -  

04:49:42 [ERROR] Specified object does not exist.

Traceback (most recent call last):
  File "/home/ajay_rana/.local/lib/python3.9/site-packages/skyplane/cli/cli.py", line 182, in cp
    transfer_pairs = generate_full_transferobjlist(
  File "/home/ajay_rana/.local/lib/python3.9/site-packages/skyplane/cli/cli_impl/cp_replicate.py", line 172, in generate_full_transferobjlist
    raise exceptions.MissingObjectException(f"No objects were found in the specified prefix {source_prefix} in {source_bucket}")
skyplane.exceptions.MissingObjectException: No objects were found in the specified prefix testskyplaneazure/containazure/ in testskyplaneazure/containazure/

❌ MissingObjectException: No objects were found in the specified prefix testskyplaneazure/containazure/ in testskyplaneazure/containazure/
Please ensure that the object exists and is accessible.

We have the data in the blob. please suggest here.
<img width="1012" alt="Screenshot 2022-10-20 at 10 22 56 AM" src="https://user-images.githubusercontent.com/29730055/196859350-4e11d119-a000-4b6e-b2c2-bae361bf98a3.png">
<img width="1172" alt="Screenshot 2022-10-20 at 10 23 51 AM" src="https://user-images.githubusercontent.com/29730055/196859511-3ffb3a44-f24b-42f5-8cab-ff004b7a4f52.png">

**Comment by sarahwooders** [comment]
@ajayra04 thanks so much for catching this and posting - could you please create an issue so we can make sure it's resolved?

**Comment by sarahwooders** [comment]
Also could you try `skyplane cp azure://testskyplaneazure/containazure gs://oradaostesr` (no trailing slash at the end of the blob) and let me know if that works?

**Reply by parasj** [reply]
@ajayra04 Actually, as an updated command please try `skyplane cp -r azure://testskyplaneazure/containazure gs://oradaostesr`. You have to pass the "-r" flag for recursive transfers.

**Comment by ajayra04** [comment]
Thanks for your prompt response @parasj @sarahwooders . 

I am still getting the same error, here is the screenshots.

Coomand Used ->  skyplane cp -r azure://testskyplaneazure/containazure gs://oradaostesr

Error ->

~$ skyplane cp -r azure://testskyplaneazure/containazure gs://oradaostesr
 _____ _   ____   _______ _       ___   _   _  _____ 
/  ___| | / /\ \ / / ___ \ |     / _ \ | \ | ||  ___|
\ `--.| |/ /  \ V /| |_/ / |    / /_\ \|  \| || |__  
 `--. \    \   \ / |  __/| |    |  _  || . ` ||  __| 
/\__/ / |\  \  | | | |   | |____| | | || |\  || |___ 
\____/\_| \_/  \_/ \_|   \_____/\_| |_/\_| \_/\____/

02:05:24 [ERROR] Specified object does not exist.

Traceback (most recent call last):
  File "/home/ajay_rana/.local/lib/python3.9/site-packages/skyplane/cli/cli.py", line 
182, in cp
    transfer_pairs = generate_full_transferobjlist(
  File 
"/home/ajay_rana/.local/lib/python3.9/site-packages/skyplane/cli/cli_impl/cp_replicate.p
y", line 172, in generate_full_transferobjlist
    raise exceptions.MissingObjectException(f"No objects were found in the specified 
prefix {source_prefix} in {source_bucket}")
skyplane.exceptions.MissingObjectException: No objects were found in the specified 
prefix testskyplaneazure/containazure in testskyplaneazure/containazure

❌ MissingObjectException: No objects were found in the specified prefix 
testskyplaneazure/containazure in testskyplaneazure/containazure
Please ensure that the object exists and is accessible.

<img width="746" alt="Screenshot 2022-10-21 at 7 35 58 AM" src="https://user-images.githubusercontent.com/29730055/197094468-073e53a9-9990-4781-9ac4-1e296eb3be87.png">
<img width="1242" alt="Screenshot 2022-10-21 at 7 39 32 AM" src="https://user-images.githubusercontent.com/29730055/197094809-7d88fbc9-01fb-4106-acef-4477641b70e2.png">
